sharing_portal/trovi.py

```diff
@@ -235,17 +235,21 @@ def create_artifact(token, artifact_id, prompt_input=False, force=False):
         print(f"This title is {len(json_data['title'])} chars (max 70)")
         print(json_data["title"])
         json_data["title"] = input("New title: ")
+    trovi_artifact = create_new_artifact(token, json_data, force)
+    artifact.trovi_uuid = trovi_artifact["uuid"]
+    artifact.save()
+    return trovi_artifact
+
+
+def create_new_artifact(token, artifact_data, force=False):
```

Comment:
Yes, the prior should be called `create_artifact_from_portal_model` or something. All of the model stuff should be deprecated since it isn't used anymore since the switch.